File: src/main/java/club/xiaojiawei/hearthstone/entity/TagChangeEntity.java
package club.xiaojiawei.hearthstone.entity;

import club.xiaojiawei.hearthstone.enums.TagEnum;
import lombok.Data;
import lombok.ToString;

/**
 * @author 肖嘉威
 * @date 2022/11/29 14:24
 */
@Data
@ToString(callSuper = true)
public class TagChangeEntity extends CommonEntity{

    private TagEnum tag;

    private String value;

}

File: Hearthstone-Script/src/main/java/club/xiaojiawei/hsscript/bean/log/TagChangeEntity.kt
package club.xiaojiawei.hsscript.bean.log

import club.xiaojiawei.hsscript.enums.TagEnum

/**
 * @author 肖嘉威
 * @date 2022/11/29 14:24
 */

class TagChangeEntity : CommonEntity() {

    var tag: TagEnum? = null

    var value: String = ""

}
